frontend/src/components/Header.tsx

```diff
@@ -296,6 +315,9 @@ const Header = ({ locale }: Props) => {
           />
         </div>
       </USWDSHeader>
+      <Snackbar close={hideSnackbar} isVisible={snackbarIsVisible}>
```

Comment:
Something to think about, but the snackbar is currently a div that has an `role` of `status`.

This would require a user to acknowledge, so should likely have a `role` of `[alert-dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/alertdialog_role) and use a `[dialog](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#accessibility) element.